py4j-java/src/test/java/py4j/instrumented/InstrClientServer.java

```diff
@@ -43,10 +43,11 @@
 public class InstrClientServer extends ClientServer {
 
 	public InstrClientServer(int javaPort, InetAddress javaAddress, int pythonPort, InetAddress pythonAddress,
-			int connectTimeout, int readTimeout, ServerSocketFactory sSocketFactory, SocketFactory socketFactory,
-			Object entryPoint, boolean autoStartJavaServer, boolean enableMemoryManagement) {
-		super(javaPort, javaAddress, pythonPort, pythonAddress, connectTimeout, readTimeout, sSocketFactory,
-				socketFactory, entryPoint, autoStartJavaServer, enableMemoryManagement);
+			int connectTimeout, int readTimeout, int createSocketTimeout, ServerSocketFactory sSocketFactory,
+			SocketFactory socketFactory, Object entryPoint, boolean autoStartJavaServer,
+			boolean enableMemoryManagement) {
+		super(javaPort, javaAddress, pythonPort, pythonAddress, connectTimeout, readTimeout, createSocketTimeout,
+				sSocketFactory, socketFactory, entryPoint, autoStartJavaServer, enableMemoryManagement);
```

Comment:
already has `connectTimeout`

```suggestion
			int connectTimeout, int readTimeout, ServerSocketFactory sSocketFactory, SocketFactory socketFactory,
			Object entryPoint, boolean autoStartJavaServer, boolean enableMemoryManagement) {
		super(javaPort, javaAddress, pythonPort, pythonAddress, connectTimeout, readTimeout, sSocketFactory,
				socketFactory, entryPoint, autoStartJavaServer, enableMemoryManagement);
```